[CUBRIDMAN-89] Revise Java SP manual in terms of Java SP server and system environments (#322)
CUBRID/cubrid-manual: ko/admin/control.rst

*   restart: 자바 저장 프로시저 서버 프로세스 재시작
*   status: 자바 저장 프로시저 서버 프로세스 상태 확인

> | 모든 명령어는 특정 데이터베이스 이름 (**[database_name]**) 을 인수로 지정할 수 있다.

Reviewer: Did you intentionally subtract the default value when databasename is not specified?

Author: I'm gonna fix the example not to omit the database name

서비스 등록
-----------

> 사용자는 임의로 데이터베이스 서버, CUBRID 브로커, CUBRID 자바 저장 프로시저 서버, CUBRID 매니저, CUBRID HA를 데이터베이스 환경 설정 파일(cubrid.conf)에 CUBRID 서비스로 등록할 수 있다. 이를 위해 cubrid.conf의 service 파라미터 값으로 각각 server, broker, manager, heartbeat를 입력하면 되며, 이들을 쉼표(,)로 구분하여 여러 개를 같이 등록할 수 있다.

Reviewer 2: Do we have to drop 'CUBRID 자바 저장 프로시저 서버' too?

Author: Thank you. I've fixed

    [service]

    # The list of processes to be started automatically by 'cubrid service start' command

>     # Any combinations are available with server, broker, manager and heartbeat.

Reviewer 2: ```suggestion
    # Any combination of  server, broker, manager and heartbeat is available.
```

Author: Thank you. but IMO, it is not the scope of this review.

Java 저장 함수/프로시저 환경 설정
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> CUBRID에서 Java 저장 함수/프로시저를 사용하기 위해서는 CUBRID 서버가 설치되는 환경에 Java Development Kit (JDK) 1.8 64bit 버전이 설치되어야 한다.    

Reviewer 3: "1.8 이상의 버전이 " is more valid ?

JDK는 다음의 경로에서 다운로드할 수 있다.

* `OpenJDK 8 <https://openjdk.java.net/projects/jdk8/>`_
* `Oracle JDK 8 <https://www.oracle.com/kr/java/technologies/javase/javase8-archive-downloads.html>`_

JRE가 이미 설치되어 있다면, 아래와 같은 명령으로 버전을 확인한다. ::

>     % java -version

Reviewer 2: Should JRE  at line 2360 be JDK?

Author: JDK has JRE. but It seems that revising the Korean manual is needed to make it clear. Thank you.

아래와 같이 명령어를 실행하여 **JAVA_HOME** 환경 변수를 설정하고 Java 실행 파일이 있는 디렉터리를 **Path** 환경 변수에 추가할 수 있다. GUI를 이용해서 환경 변수를 설정하는 방법은 JDBC 설치 및 설정을 참고한다.

* JDK 1.8 환경 변수를 설정한 예 ::

    % set JAVA_HOME=C:\jdk1.8.0
    % set PATH=%PATH%;%JAVA_HOME%\jre\bin\server

SUN의 Java 가상 머신을 사용하지 않고 다른 벤더의 구현을 사용하는 경우를 포함하여 명시적으로 Java 가상 머신 (JVM)의 경로를 지정하려면 **jvm.dll** 파일의 경로를 **JVM_PATH** 환경 변수에 추가한다.
CUBRID는 먼저 **JVM_PATH** 변수에서 **jvm.dll** 파일의 경로를 찾는다. **JVM_PATH** 가 설정되지 않았거나 파일을 로드할 수 없는 경우 위에서 설명한 **JAVA_HOME** 변수에서 **jvm.dll** 을 찾는다.

*   **JVM_PATH** 환경 변수를 설정한 예 ::
    
    % set JVM_PATH=C:\jdk1.8.0\jre\bin\server\libjvm.dll

**Linux/Unix 환경**

CUBRID는 Linux/Unix 환경에서 **libjvm.so** 파일을 로딩하여 Java 가상 머신을 실행시킨다. CUBRID는 먼저 **LD_LIBRARY_PATH** 환경 변수에서 **libjvm.so** 파일을 찾아 로딩한다. 만약 찾지 못하면 **JAVA_HOME** 환경 변수를 이용하여 찾는다. 리눅스의 경우 glibc 2.3.4 이상만 지원되며, 아래는 리눅스 환경 설정 파일(예: **.profile**, **.cshrc**, **.bashrc**, **.bash_profile** 등)에 환경 변수를 설정하는 예이다.

*   bash 셸에서 JDK 1.8 환경 변수를 설정한 예 ::

    % JAVA_HOME=/usr/java/jdk1.8.0
    % LD_LIBRARY_PATH=$JAVA_HOME/jre/lib/amd64:$JAVA_HOME/jre/lib/amd64/server:$LD_LIBRARY_PATH
    % export JAVA_HOME
    % export LD_LIBRARY_PATH

> *   csh 셸에서 JDK 1.8 환경 변수를 설정한 예 ::

Reviewer 4: csh? bash?

Author: csh is right. I think GitHub file changes make you confuse because of mis-align
